modules/g/Geniate/data-source/aws.json

```diff
@@ -0,0 +1,10 @@
+{
+  "versions": [
+    {
+      "version": "0.0.2"
+    },
```

Comment:
this version does not exist at the moment

Reply:
```suggestion
```

Reply:
Hi @barshopen the module seems to have a 0.0.2 tag, that's why it got picked up by the automated process: https://github.com/Geniate/terraform-aws-data-source/tree/0.0.2

You may want to delete this tag for the process to not pick it up.

Reply:
Removed, thanks